tools/bazar/controllers/ApiController.php

```diff
@@ -39,26 +39,31 @@ public function getForm($formId)
     }
 
     /**
-     * @Route("/api/fiche/{formId}", methods={"GET"})
+     * @Route("/api/fiche/{formId}/{output}/{selectedEntries}", methods={"GET"})
      */
-    public function getAllEntries($formId)
+    public function getAllEntries($formId, $output = null, $selectedEntries = null)
     {
-        if( strpos($_SERVER['HTTP_ACCEPT'], 'application/ld+json') !== false) {
-            return $this->getAllSemanticEntries($formId);
-        }
 
-        $entries = $this->getService(EntryManager::class)->search([ 'formsIds'=>$formId ]);
+        $entries = $this->getService(EntryManager::class)->search([
+            'formsIds'=>$formId,
+            'selectedEntries'=>$selectedEntries
```

Comment:
```suggestion
            'queries'=>!empty($selectedEntries) ? ['id_fiche' => $selectedEntries]: []
```
ceci devrait fonctionner, sans avoir besoin de modifier EntryManager.
Qu'en dis-tu @mrflos ?
J'ai testé ça fonctionne et ça évite de complexifier EntryManager->search

Reply:
ok, on regarde avec @nils-van-zuijlen pour modifier en conséquence, je ne pensais pas que queries permettait le multicritère

Reply:
Oui, effectivement, ça fonctionne aussi comme ça, et comme l'a dit un grand sage, "chaque ligne en plus est une chance de plus pour de erreurs"